olm/operator-source.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: cert-utils-operator
+  namespace: openshift-marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
+  registryNamespace: raffaelespazzoli
```

Comment:
Please update with the cop namespace

Reply:
again I'll do if I have the credentials to use it. otherwise it's better to leave it like that which makes it clear that each dev will have to use its own repo. keep in mind this is just for manual testing.